cmd/gtk/dialog_wallet_create_address.go

```diff
@@ -21,27 +20,39 @@ func createAddress(ww *widgetWallet) {
 	addressLabel := getEntryObj(builder, "id_entry_account_label")
 
 	addressTypeCombo := getComboBoxTextObj(builder, "id_combo_address_type")
-	addressTypeCombo.Append(wallet.AddressTypeBLSAccount, "Account")
+	addressTypeCombo.Append(wallet.AddressTypeEd25519Account, "ED25519 Account")
+	addressTypeCombo.Append(wallet.AddressTypeBLSAccount, "BLS Account")
 	addressTypeCombo.Append(wallet.AddressTypeValidator, "Validator")
 
 	addressTypeCombo.SetActive(0)
 
-	getButtonObj(builder, "id_button_ok").SetImage(OkIcon())
-	getButtonObj(builder, "id_button_cancel").SetImage(CancelIcon())
-
 	onOk := func() {
 		walletAddressLabel, err := addressLabel.GetText()
 		fatalErrorCheck(err)
 
 		walletAddressType := addressTypeCombo.GetActiveID()
 		fatalErrorCheck(err)
 
-		if walletAddressType == wallet.AddressTypeBLSAccount {
+		switch walletAddressType {
+		case wallet.AddressTypeEd25519Account:
+			passwordFetcher := func(wlt *wallet.Wallet) (string, bool) {
+				if *passwordOpt != "" {
+					return *passwordOpt, true
+				}
+
+				return getWalletPassword(wlt)
+			}
+
+			password, ok := passwordFetcher(ww.model.wallet)
```

Comment:
@Ja7ad why don't we use it like:
```go
password, ok := getWalletPassword(wlt)
		if ok {
			_, err = ww.model.wallet.NewEd25519AccountAddress(walletAddressLabel, password)
		}
```

Reply:
Not good, ED is default type and always open create account modal show password.

Reply:
What do you mean default type?